objc/UIViewController+PromiseKit.m

```diff
@@ -6,11 +6,13 @@
 #import <UIKit/UINavigationController.h>
 #import <UIKit/UIImagePickerController.h>
 #import "UIViewController+PromiseKit.h"
+@import MessageUI.MFMessageComposeViewController;
+@import MessageUI.MFMailComposeViewController;
```

Comment:
Can't @import as this imposes framework requirements on our end users. Since objc is dynamic, we don’t need to, hence we were using `NSClassFromString`.